corehq/toggles.py

```diff
@@ -93,8 +93,8 @@ def index(self):
 class StaticToggle(object):
 
     def __init__(self, slug, label, tag, namespaces=None, help_link=None,
-                 description=None, save_fn=None, always_enabled=None,
-                 always_disabled=None, enabled_for_new_domains_after=None,
+                 description=None, save_fn=None, always_enabled=set(),
+                 always_disabled=set(), enabled_for_new_domains_after=None,
```

Comment:
default values should never be mutable: https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments

Reply:
Thanks, missed that.